client/src/App.css

```diff
@@ -14,13 +14,15 @@
   --shadow: #959da533 0px 8px 24px; /* box-shadow: var(--shadow); */
   --green-success: #aad0a3;
   --red-error: #dd4343;
-  --text-link: #90caf9;
+  --text-link: #4471d9;
   --text-dark: #20002e;
   --text-light: #f3f3f3;
   --text-muted: #696969;
   --navbar-bg: #ebebeb;
   --navbar-underline: #e2e2e2;
   --page-bg: #f6f3fb; /* very light purple */
```

Comment:
also james, would it be possible to use this for the background colour of the page? 
or if you tried it already and it doesn't look good feel free to get rid of this!

Reply:
Yep! You can apply the background color to the body (and maybe the html tag if needed) in App.css

Reply:
you can submit a commit on this PR, or I can do it tomorrow

Reply:
yep! i can do that 👍